api/src/lib.rs

```diff
@@ -1570,33 +1606,27 @@ where
     }
 
     {
-        // Verification tokens legacy endpoint. Remove for API v3.
-        let mut verify_tokens_legacy =
-            tide::with_state(State::new(verify_db.clone(), token_db.clone()));
-
-        verify_tokens_legacy
-            .at(":id")
-            .with(ensure_vault_id)
-            .with(cors.clone())
-            .options(option_ok)
-            .get(get_verification_status)
-            .post(create_verification_token)
-            .put(verify_verification_token);
-
-        api.at("verify_tokens").nest(verify_tokens_legacy);
-
         // Verification tokens
         let mut verify =
             tide::with_state(State::new(verify_db.clone(), token_db.clone()));
 
         verify
             .at("entries/:id")
             .with(ensure_vault_id)
-            .with(cors)
             .options(option_ok)
-            .get(get_verification_status)
-            .post(create_verification_token)
+            .with(cors)
```

Comment:
I'm going to test this without the rust CORS handling because we have set up CORS in digital ocean since i wrote the rust CORS stuff originally.

Reply:
the CORS stuff needs to be there if you run everything on localhost, because of localhost:3000 api server and the js running out of localhost:8000. I'm not really sure what to do about this.

Reply:
this is fixed by the commit i made switching the order of .options <-> with(cors)